test/functional/fixtures/runner/uncaught-js-errors/uncaught-js-errors-test.js

```diff
@@ -0,0 +1,73 @@
+var expect    = require('chai').expect;
+var errorType = 'Uncaught JavaScript error Uncaught Error:';
+
+describe('Uncaught js errors.', function () {
+    it('Should fail when are no handler', function () {
+        return runTests('no-handler/test.js', null, { shouldFail: true })
+            .catch(function (err) {
+                expect(err).to.contains(errorType);
+            });
+    });
+    it('Should fail when handler returns undefined', function () {
+        return runTests('handler-returns-undefined/test.js', null, { shouldFail: true })
+            .catch(function (err) {
+                expect(err).to.contains(errorType);
+            });
+    });
+
+    it('Should fail when iframe\'s source is a page with handler which returns undefined', function () {

```

Comment:
I suppose we can simplify this
`Should fail when iframe's onerror handler returns undefined`